Readme.md

```diff
@@ -17,6 +17,7 @@ Xamarin creates and maintains Xamarin.iOS bindings for the Google APIs for iOS L
 | [Xamarin.Firebase.iOS.Database][F.Database.Name]                             | [6.0.0.2][F.Database.Package]                |
 | [Xamarin.Firebase.iOS.DynamicLinks][F.DynamicLinks.Name]                     | [4.0.1.2][F.DynamicLinks.Package]            |
 | [Xamarin.Firebase.iOS.InstanceID][F.InstanceID.Name]                         | [4.3.0.0][F.InstanceID.Package]              |
+| [Xamarin.Firebase.iOS.Functions][F.Functions.Name]                           | [6.4.2.0][F.Functions.Package]               |
```

Comment:
What version should the nuget package have? I don't understand the current version schema

Reply:
Firebase handles two versions. One for the whole set of components that it offers (global version) and the other one for each component. For example, [Firebase for iOS 6.15][1] offers:

* A/B Testing 3.1.2.0
* AdMob 7.53.1.0
* Analytics 6.2.1.0
* Auth 6.4.2.0 ...

and so on. So the **InAppMessaging** version that correspond to 6.15.0 global version is **0.16.0**.

[1]: https://github.com/CocoaPods/Specs/blob/master/Specs/0/3/5/Firebase/6.15.0/Firebase.podspec.json

Reply:
So, I suggest to change the name to its official Firebase name that is `Cloud Functions`

```suggestion
| [Xamarin.Firebase.iOS.CloudFunctions][F.CloudFunctions.Name]                           | [2.5.1.0][F.CloudFunctions.Package]               |
```